src/common/components/profile-edit/index.tsx

```diff
@@ -87,12 +96,12 @@ export default class ProfileEdit extends BaseComponent<Props, State> {
         }).then((account) => {
 
             // reload page to refresh profile image
-            if (activeUser.data.__loaded && activeUser.data.profile?.profile_image !== account.profile?.profile_image) {
-                setTimeout(() => {
-                    window.location.reload();
-                }, 500);
-                return;
-            }
+            // if (activeUser.data.__loaded && activeUser.data.profile?.profile_image !== account.profile?.profile_image) {
```

Comment:
reload not needed anymore right? we can remove this part instead of comment if that's the case